App/SB/views/UserProfile/index.js

```diff
@@ -38,14 +36,14 @@ class UserProfile extends React.PureComponent {
       ),
       headerRight: (
         <HeaderButtons>
-          <Item 
+          <Item
             title='Avatar'
             buttonWrapperStyle={{marginLeft: 11, marginRight: 11}}
             ButtonElement={
               <Avatar
-                width={32} 
-                height={32} 
-                uri={params.avatarUrl} 
```

Comment:
Just set my editor to trim trailing whitespace, should get rid of all this PR noise with us.

Reply:
excellent